lib/ecto/changeset.ex

```diff
@@ -680,6 +680,32 @@ defmodule Ecto.Changeset do
         end
       end
 
+  ## Partial changes for many-style associations
+
+  While the introductionary sentences for this function talk about `cast_assoc/3`
```

Comment:
I would open with something clearer like “if you’ve preloaded an association, that will confine the behavior of `cast_assoc` to the records you’ve preloaded.